### ISSUE TITLE
Cheat Engine Driver failed to load due to signing issues 

### ISSUE THREAD

**Issue by RavenNexuss** [cheat-engine/cheat-engine#2715]
--> **DBK32 error
The driver failed to load due to signing issues. If you have secure boot enabled in your BIOS, set it to "Other OS" or disable it. Alternatively, boot with driver signing policy disabled, or sign the driver yourself**

It needs me to turn off Driver Signature Enforcement, but Easy Anticheat needs it on, and when I use a bootloader such as EFI, the anti-cheat also detects that kernel patch protection has been turned off including that they is a patched bootloader. I don't know how to sign a driver myself nor do I have the money to get a CA certificate. I need a way basically to turn off DSE and KPP on runtime and if its not possible, give me a solution. Please I need help, I do I fix this?

@cheat-engine <3 <---
![Screenshot 2023-10-15 154852](https://github.com/cheat-engine/cheat-engine/assets/147989903/06e19442-d2d1-43d8-af5a-ca1355c75852)

**Comment by cheat-engine** [comment]
Nope, some idiot managed to bypass CE's protection from being used, so i'm not going to fix this ever again

**Comment by RavenNexuss** [comment]
> Nope, some idiot managed to bypass CE's protection from being used, so i'm not going to fix this ever again. Just buy a class 3 CA certificate for yourself and sign it with that

But I am broke :/, I don't have any money at all. like any chance you can give me the thing that made it work before it got bypassed privately, I beg you please, and secure boot is off. @cheat-engine

**Comment by RavenNexuss** [comment]
Hello?

**Comment by RavenNexuss** [comment]
And in my currency to buy a Certificate is about 2k, I can't afford that. Please @cheat-engine

**Comment by bb33bb** [comment]
> Nope, some idiot managed to bypass CE's protection from being used, so i'm not going to fix this ever again. Just buy a class 3 CA certificate for yourself and sign it with that
> 
> or at least turn of secure boot and use windows 10

this answered my question, I wondered the root problem of win 11 not function well. 
Now I should go to use win10.

**Comment by RavenNexuss** [comment]
> > Nope, some idiot managed to bypass CE's protection from being used, so i'm not going to fix this ever again. Just buy a class 3 CA certificate for yourself and sign it with that
> > or at least turn of secure boot and use windows 10
> 
> this answered my question, I wondered the root problem of win 11 not function well. Now I should go to use win10.

So it now works when you turned back to windows 10?

**Comment by p3441469485** [comment]
> > > Nope, some idiot managed to bypass CE's protection from being used, so i'm not going to fix this ever again. Just buy a class 3 CA certificate for yourself and sign it with that
> > > or at least turn of secure boot and use windows 10
> > 
> > 
> > this answered my question, I wondered the root problem of win 11 not function well. Now I should go to use win10.
> 
> So it now works when you turned back to windows 10?

You can use older versions of Windows, such as Windows 2021LTSC, 1809, 1909..., so you can load the driver with some leaked expired certificate signatures, but you still can't bypass the EAC. You need to modify the driver source code to bypass the EAC.

**Comment by SpiderZombie** [comment]
please , "i don't know what you did,you didn't crashmbut you also didn't load dbvm"

**Comment by cheat-engine** [comment]
is vmdisk.img present ?

**Comment by supundhanujaya** [comment]
[vmdisk_[unknowncheats.me]_.zip](https://github.com/user-attachments/files/16861042/vmdisk_.unknowncheats.me._.zip)
@SpiderZombie copy this vmdisk to the bin folder
